.vscode/launch.json

```diff
@@ -69,7 +73,8 @@
 			"preLaunchTask": "npm: watch",
 			"env": {
 				"VST_DISABLE_COVERAGE": "true",
-				"VSCODE_SINGLE_TEST": "${relativeFile}"
+				"VSCODE_SINGLE_TEST": "${relativeFile}",
+				"VSCODE_REDHAT_TELEMETRY_DEBUG": "true"
```

Comment:
Same there

Reply:
This is required to make sure telemetry data sent during development is sent to the test bucket

Reply:
But we use that configuration to launch tests from vscode. Not sure that we need to sent telemetry data during running tests.

Reply:
Yes but not sure you can control that so this is a setting that make sure you're doing it right

Reply:
Hm, it seems that they have mistake in doc,
the only thing, which I found, `VSCODE_REDHAT_TELEMETRY_DEBUG` is controls logs for telemetry:
https://github.com/redhat-developer/vscode-redhat-telemetry/blob/main/src/utils/logger.ts
and does nothing with sending telemetry events. In such case we can add back `VSCODE_REDHAT_TELEMETRY_DEBUG ` in our configuration.